src/components/Sidebar.js

```diff
@@ -1,13 +1,72 @@
-import { Link } from 'react-router-dom';
+import { useState } from 'react';
+import { Link, useLocation } from 'react-router-dom';
+import { Icon } from '@iconify/react';
 
 export default function Sidebar() {
+  const [open, setOpen] = useState(true);
```

Comment:
I'm sure you noticed that this state doesn't persist across page changes. Two ways I can think of dealing with this: 
- Store this local state in Recoil state instead, but this feels a bit hacky
- Reorganize the PageWrapper logic, hoisting it from our individual `pages/*` files into the index file. For example, the below component nesting should work. 

I thinkkkk I lean towards the second, but I could be swayed either way. 
![image](https://user-images.githubusercontent.com/5703736/213273736-81427a59-064b-49bf-8579-c1447ecd6d2a.png)